cacerts/java.bzl

```diff
@@ -27,7 +27,8 @@ def _impl(ctx):
 set -ex
 docker load -i {0}
 # Install the certs in the builder image.
-cid=$(docker run -d {1} sh -c "apt-get update && apt-get install -y -q ca-certificates-java")
+# ln: the default non-interactive shell is dash, which interferes with the post install script.
+cid=$(docker run -d {1} sh -c "ln -sf bash /bin/sh && apt-get update && apt-get install -y -q ca-certificates-java")
```

Comment:
This is odd as from poking in the .deb, the `postinst` file has a `#!/bin/bash` as its shebang.

Reply:
Oh, maybe it's one of the prereqs.

Reply:
It uses `!#/bin/sh`, but it has `==` (double equals) in a conditional that bash tolerates but dash doesn't.

Reply:
I mean, there may be a script with the bash shebang, but that is not the script causing an issue.

Reply:
The error we see is from the `jks-keystore` script at line 56.

```
Running hooks in /etc/ca-certificates/update.d...

/etc/ca-certificates/update.d/jks-keystore: 56: [: amd64: unexpected operator
done.
```

The version of the package being installed is 20170929~deb9u1.

```
# dpkg -l | grep certif
ii  ca-certificates              20161130+nmu1+deb9u1  all          Common CA certificates
ii  ca-certificates-java         20170929~deb9u1       all          Common CA certificates (JKS keystore)
```

`jks-keystore.hook.in` from the matching date 20170929 has `!#/bin/sh` and uses `==` at line 56: https://salsa.debian.org/java-team/ca-certificates-java/blob/debian/20170929/debian/jks-keystore.hook.in#L56